src/gluonnlp/data/transforms.py

```diff
@@ -988,10 +1004,261 @@ def convert_tokens_to_ids(self, tokens):
         return self.vocab.to_indices(tokens)
 
 
+class BERTTransform(object):
+    """BERT style data transformation.
+
+    Parameters
```

Comment:
As we move code to API we need more unit tests to make sure it is not broken in the future. Do you mind adding one?

Reply:
Yes, this is supposed to be. What I want is to add a unit test after the API code meets the requirements.